src/main/java/org/mitre/synthea/world/geography/Location.java

```diff
@@ -437,6 +438,18 @@ public static String getAbbreviation(String state) {
     return stateAbbreviations.get(state);
   }
 
+  /**
+   * Get the state for an abbreviation.
+   * @param abbreviation State abbreviation. e.g. "MA"
+   * @return state name. e.g. "Massachusetts"
+   */
+  public static String getStateNameFromAbbreviation(String abbreviation) {
+    for(Entry<String, String> entry : stateAbbreviations.entrySet()) {
+      if (entry.getValue().equals(abbreviation)) return entry.getKey();
+    }
+    return null;
+  }
+
```

Comment:
This method duplicates:
https://github.com/synthetichealth/synthea/blob/2cc0a62b07e86bcd7019f177731201d772df233d/src/main/java/org/mitre/synthea/world/geography/Location.java#L458-L470